bin/pullRawDataFromDS.sh

```diff
@@ -209,8 +205,22 @@ else
 		gsProject="$(basename "${gsProject}")"
 		if [ -e "/groups/${GROUP}/${TMP_LFS}/logs/${gsProject}/${gsProject}.processGsRawData.finished" ]
 		then
-			log4Bash 'WARN' "${LINENO}" "${FUNCNAME:-main}" '0' "${gsProject} already processed, no need to transfer the data again"
+			log4Bash 'WARN' "${LINENO}" "${FUNCNAME:-main}" '0' "${gsProject} already processed, no need to transfer the data again."
 		else
+			#
+			# Check if gsProject is supposed to be complete (*.finished present).
+			#
+			gsProjectUploadCompleted='false'
+			# shellcheck disable=SC2029
+			if ssh "${HOSTNAME_DATA_STAGING}" "find \"/groups/${GROUP}/${SCR_LFS}/${gsProject}/\" -mindepth 1 -maxdepth 1 -name '*.finished'"
+			then
+				gsProjectUploadCompleted='true'
+			fi
+			#
+			# Rsync everything but the .finished file: may be imcompletely uploaded project,
+			# but we already rsync everything we've got so far.
+			#
+			log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "Rsyncing everything but the .finished file for ${gsProject} ..."
```

Comment:
log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "Rsyncing everything but the .finished file for ${gsProject} ..." moet denk ik gelden voor alleen de gsProjectUploadCompleted='false' projecten. nu geldt het ook voor gsProjectUploadCompleted='true' prjecten. of zie ik dat verkeerd?

Reply:
Moet voor beide: of er nu al wel een *.finished is of niet: alles wat er al wel is halen we alvast binnen. Was het incompleet dan komt de rest bij een volgende keer dat de cronjob langs komt.
Alleen als de *.finished er al was voordat we alles behalve de *.finished gingen syncen mogen we ervan uitgaan dat GS klaar was met de upload en dan syncen we ook de *.finished.